evadb/udfs/chatgpt.py

```diff
@@ -97,21 +97,26 @@ def completion_with_backoff(**kwargs):
         results = []
 
         for prompt, query in zip(prompts, queries):
-            if prompt != "None":
-                query = prompt + ": " + query
+            # if prompt != "None":
+            #     query = prompt + ": " + query
 
             params = {
                 "model": self.model,
                 "temperature": self.temperature,
                 "messages": [
                     {
                         "role": "user",
-                        "content": query,
+                        "content": f"Context to answer the question : {query}",
+                    },
+                    {
+                        "role": "user",
+                        "content": f"Answer the question based on context : {prompt} Answer only from the context, don't generate responses on your own",
                     }
                 ],
             }
 
             response = completion_with_backoff(**params)
+            print("Response : ", response.choices[0].message.content)
```

Comment:
is this a debug message or is this needed for the workflow? if not, we probably don't need it

Reply:
Was put in for debugging, removed it. Thanks for pointing it out.